dist/react-number-format.js

```diff
@@ -202,50 +202,8 @@ return /******/ (function(modules) { // webpackBootstrap
 	      }
 	    }
 	  }, {
-	    key: 'formatWithPattern',
```

Comment:
I assume this file is generated, and actually should not be included into the repo.